Classes/Classes/miniphonedir.cpp

```diff
@@ -0,0 +1,101 @@
+#include<iostream> 
+
+using namespace std; 
+
+  
+
+class UserData { 
+
+private: 
+
+    string name; 
+
+    string last_name; 
+
+    string email; 
+
+    string phone_number; 
+
+    /*Method*/ 
+
+public: 
+
+    void set_name(string name_temp); 
```

Comment:
Use two different methods, one for first name and one more for last name
It's also a good practice to use const string& as the function argument instead of string to avoid calling the copy constructor
void set_first_name(const std::string& name_temp);

Reply:
What does the copy constructor does? Does it stores a new value in the stack?